### PR TITLE
docs: data seeder for current models

### PR DESCRIPTION
**Pull request by ashleynhs**
## Overview

Added a data seeding mechanism for dev environment using TypeORM and typeorm-seeding. The seeder generates consistent sets of data so that all devs have the same development data set to work with.

## Changes Made

Added factories for multiple models (UserModel, PostModel, FeedbackModel, UserReviewModel, ReportModel, RequestModel) to generate consistent data.

## Test coverage

No test coverage

## Running Script

npm run db:seed

### Note

The seeding script checks the environment to ensure it only runs in the dev environment. If the environment variable IS_PROD is set to "true", the seeding process will be skipped to prevent accidental execution in prod.

### CODE REVIEW

File: ormconfig.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { SnakeNamingStrategy } from 'typeorm-naming-strategies';
+//import { SnakeNamingStrategy } from 'typeorm-naming-strategies';
```
Comment (Aayush-Agnihotri) [inline ormconfig.ts:1]: delete instead of commenting to keep the code clean

File: ormconfig.ts [hunk 2]
```diff
@@ -11,10 +11,12 @@ module.exports = {
       'src/models/*.ts',
    ],
    synchronize: false,
-   namingStrategy: new SnakeNamingStrategy(),
+   //namingStrategy: new SnakeNamingStrategy(),
```
Comment (Aayush-Agnihotri) [inline ormconfig.ts:14]: same here

**Review by Aayush-Agnihotri — APPROVED** [review]
LGTM, great job! We should also revamp our tests to use these new factories instead too

**Review by Daniel-jw — APPROVED** [review]
LGTM

**Pull request merged**